thangs-blender-addon_updater/thangs-blender-addon_updater_status.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
We probably shouldn't include this file.  Can you delete it and then make sire that we ignore it in the `.gitignore`?